### ISSUE TITLE
Are v3 docs available anywhere?

### ISSUE THREAD

**Issue by jimmymorris** [style-dictionary/style-dictionary#1310]
Given that the v4 requires quite a few breaking changes, it may not be feasible for teams to upgrade to v4 right away.

Is there any way we can still have access to the v3 docs or should we just checkout v3 tag and run it locally?

**Comment by Yeom-JinHo** [comment]
@jimmymorris I think this is the v3 docs: https://amzn.github.io/style-dictionary/#/README

**Comment by jorenbroekema** [comment]
yup that's it. We may redirect that one to the v4 docs eventually but we should have the v3 docs available as well before we do so. 
Maybe a v3.styledictionary.com subdomain with a link in the v4 docs pointing to it.

**Comment by jorenbroekema** [comment]
Closing question issue, considered as answered.

**Comment by SalimBensiali** [comment]
https://amzn.github.io/style-dictionary/#/README now leads to a 404 not found. Are the v3 docs still available anywhere?

**Comment by zachbwh** [comment]
@SalimBensiali You can find them on the wayback machine [here](https://web.archive.org/web/20250419023213/https://amzn.github.io/style-dictionary/#/config?id=attributes). Not ideal though, they should be brought back :/

**Comment by BitwiseAndrea** [comment]
Please bring these docs back!!

**Comment by jorenbroekema** [comment]
I pushed and fixed the v3 branch and deployed the v3 docs at a temporary address: https://styledictionary-v3.netlify.app/#/

Reopening for now. Will close it once it's available as a subdomain on the styledictionary.com domain, and it's easy to find and navigate to from the current docs.

@dbanksdesign maybe something you can help with? At least the auto deploying v3 branch to subdomain part? Then I can do the 2nd part of making it easy to find in the current docs

**Comment by dbanksdesign** [comment]
https://v3.styledictionary.com/ is up and is auto-deploying from the v3 branch.
